doc/runbooks/application-recovery.md

```diff
@@ -0,0 +1,21 @@
+## Application Recovery
+
+With reference to [ADR 0027](../architecture/decisions/0027-recovery-and-rollback-strategy.md) for rollback strategy
+
+### Code release rollback
+
+Use these steps to roll back to the previous application version:-
+
+1. Go into the CodeDeploy in the AWS console.
+2. Find the previous deployment and click into it.
+3. Click retry deployment, this will cause the deployed version to be rolled back.
+4. Fix and merge new code into git.
+
```

Comment:
Have we done a trial run of this?

Reply:
I have yep, or at least the redeployment of old code